meson_cpu/x86/meson.build

```diff
@@ -60,7 +60,7 @@ FMA3 = mod_features.new(
   test_code: files(source_root + '/numpy/distutils/checks/cpu_fma3.c')[0]
 )
 AVX2 = mod_features.new(
-  'AVX2', 25, implies: F16C, args: '-mavx2',
+  'AVX2', 25, implies: F16C, args: '-march=skylake',
```

Comment:
Skylake is considerably more than AVX2. To enable HWY_AVX2, -maes -march=haswell are sufficient.